sherpa/ui/utils.py

```diff
@@ -2466,9 +2466,9 @@ def set_iter_method(self, meth):
 
         This is a chi-square statistic where the variance is computed
         from model amplitudes derived in the previous iteration of the
-        fit. This 'Iterative Weighting' ([1]_) attempts to remove
-        biased estimates of model parameters which is inherent in
-        chi-square statistics ([2]_).
+        fit. This 'Iterative Weighting' ([1]) attempts to remove
+        biased estimates of model parameters which is
+        `inherent in chi-square statistics <http://iraf.net/irafhelp.php?val=sfit>`_,
```

Comment:
Unfortunately I think this is now a dead link (iraf.net still exists but I can't seem to get any usable link to the sfit command or whatever this link was meant to take us to).

Reply:
Now that you pointed that out, I tried the waybackmachine, but that does not help because the link supposedly triggers a php script which they don't archive. I suggest I just remove the link and leave the text stand on its own?

Reply:
@anetasie - do you think it's have the text (modulo typos, case, and quotes)

> This iterative weighting attempts to remove biased estimates of model parameters which is inherent to chi-square statistics.

or does there need to be some way to reference this (and if so, what should it be)?

My feeling is we don't need to, but this could just be me...

Reply:
@DougBurke  the IRAF documentation is now on readthedocs: https://iraf.readthedocs.io/en/latest/tasks/noao/imred/specred/sfit.html

Reply:
So, looking at the current `sfit` documentation I don't think it's a useful reference for the statistical point. I've just added a comment for a different location to update the iraf.net link, so we can change it to

```suggestion
        inherent in chi-square statistics,
```

(although if we do this we probably want to re-indent the whole paragraph) and then add an issue to work out what we actually want to say here (from a statistical point of view, as this PR is meant to be a "mechanical" change rather than a substantive one).

Reply:
I understand that the link was mainly to the method of iterative fitting, not related to bias in fitting or chi2 statistics. The wording is a bit confusing. In the 'iterative sigma clipping' the outliers are removed at each iterative step. Perhaps instead of 
'bias' one should refer to outliers - point 'n-sigma' away from the best fit model.

Reply:
I don't think I know where this references are located in RTD page.

Reply:
@anetasie - this is part of the `set_iter_method` docs, so https://sherpa--1813.org.readthedocs.build/en/1813/ui/api/sherpa.ui.set_iter_method.html

Reply:
I'm just removing the `inherit in chi-square statistics` part completely, so the paragraph will become

        This is a chi-square statistic where the variance is computed
        from model amplitudes derived in the previous iteration of the
        fit. This 'Iterative Weighting' ([1]) attempts to remove
        biased estimates of model parameters.